### DISCUSSION TITLE
Which prune method I need to sync for getting address balance?

### DISCUSSION THREAD

**Discussion by 0xisakov** [erigontech/erigon#7298] [Q&A]
Hello guys. I like this software so much, you are doeing great job. 
Please help me to setup it.  Tell which prune flags I need to use to minimize blockchain size, and to save ability get address balance using rpc_method `eth_getBalance`, for getting only latest address balance?

Right now I'm using `--prune htc` but `eth_getBalance` returning `{"jsonrpc":"2.0","id":1,"result":"0x0"}` on a address which is not empty.

Please tell me what I am doeing wrong.
Here is my erigon parameters:
``` erigon --prune htc --datadir=erigon_dir --chain=mainnet --torrent.download.rate=20mb --http --ws```

**Comment by AskAlexSharov** [comment]
any method
likely you are not synced yet: show logs
